ts/features/fci/store/actions/index.ts

```diff
@@ -47,6 +47,20 @@ export const fciSigningRequest = createAsyncAction(
   "FCI_SIGNING_FAILURE"
 )<CreateSignatureBody, void, NetworkError>();
 
+/**
+ * asycn action to download file
+ */
+export const fciDownloadPreview = createAsyncAction(
+  "FCI_DOWNLOAD_PREVIEW_REQUEST",
+  "FCI_DOWNLOAD_PREVIEW_SUCCESS",
+  "FCI_DOWNLOAD_PREVIEW_FAILURE",
+  "FCI_DOWNLOAD_PREVIEW_CANCEL"
+)<{ url: string }, { path: string }, NetworkError, void>();
+
+export const fciDownloadPreviewCancel = createStandardAction(
```

Comment:
Isn't the cancel already defined on the action above?

Reply:
Yep. It's my fault already fixed in the #4239 I update this one. Thx!